exporter/diagnostic_data_collector_test.go

```diff
@@ -60,6 +64,9 @@ mongodb_oplog_stats_wt_transaction_update_conflicts 0` + "\n")
 		"mongodb_oplog_stats_wt_btree_fixed_record_size",
 		"mongodb_oplog_stats_wt_transaction_update_conflicts",
 	}
-	err := testutil.CollectAndCompare(c, expected, filter...)
+	reg := prometheus.NewRegistry()
```

Comment:
Can we use `NewPedanticRegistry` there for additional checks?

Reply:
We used it, but it made this test unstable. I think it's because we describe by collect. When system makes first collect to describe metrics there is possibility to not have some metrics which exist in next collect calls.

Reply:
Then that's a real problem that requires a separate ticket for the future

Reply:
Ok, then I change back to NewRegistry till the next ticket.